apps/rush/src/actions/RushCommandLineParser.ts

```diff
@@ -25,8 +24,8 @@ import ScanAction from './ScanAction';
 import Telemetry from '../utilities/Telemetry';
 
 export default class RushCommandLineParser extends CommandLineParser {
```

Comment:
>RushCommandLineParser [](start = 21, length = 21)

Could you see what's involved to add a unit test that confirms that Rush can print out the command-line help?

It could be something like this:

- Change the current working directory to a folder with an unparseable "rush.json" file in it.
- Since I never got around to removing this crappy "process.exit(1)", invoke a separate NodeJS instance on './lib/rush.js'
- Confirm that the process exit code is 0